.github/workflows/all-prod-tn.yml

```
@@ -0,0 +1,91 @@
+  push:
+    branches:
+      - all-1.3-tn-prod-hotfix
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment: all-prod-tn
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)